packages/blocks/code/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hashintel/block-code",
-  "version": "0.2.1",
+  "version": "0.2.0",
```

Comment:
Lowering the version for consistency with other blocks. The versions are nominal ATM anyway.